plugins/timelib/timelib.cpp

```diff
@@ -657,6 +659,23 @@ TIMELIB_API void TIMELIB_CALL tlSecondsToString(size32_t &__lenResult, char* &__
 
 //------------------------------------------------------------------------------
 
+TIMELIB_API __int64 TIMELIB_CALL tlStringToSeconds(size32_t lenS, const char* s, const char* fmtin, bool is_local_time)
```

Comment:
I think this would probably be better to leave the strptime code where it was and add this function into stringlib.cpp, otherwise the code will be compiled into two shared objects.  The function can still be available from Std.Date.

@richardkchapman what do you think?

Reply:
The problem I was dealing with was that tlStringToSeconds relies on non-trivial code from both stringlib and timelib.  strptime and tlMkTime (which contains Linux and Windows versions), respectively.  I originally wanted to migrate strptime and all functions relying it to timelib, but Richard talked me out of it.

Reply:
Looks to me as though the code you want to share (simple_strptime) is not currently exported from stringlib, meaning that no-one can be calling it from existing ECL code. So it would be ok to move that code into a shared location (e.g. eclrtl) and to have both these plugins call it there.

Reply:
I will move strptime and its related functions to eclrt.  That will remove the common code currently in place.